tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java

```diff
@@ -47,7 +47,7 @@ public void setup() {
      */
     @Test
     public void testCircuitBreaker() {
-        int[] expectedStatusCodes = new int[] { 200, 200, 502, -1, -1, 200, 200, 200 };
```

Comment:
Please put a comment on what this -1 means

Reply:
These status codes haven't been used anymore.